src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -50,3 +50,39 @@ def test_get_self_cal_last_date_and_time():
 
 def test_self_cal(session):
         session.self_cal()
+
+
+def test_standard_waveform(session):
+    session.output_mode = nifgen.OutputMode.NIFGEN_VAL_OUTPUT_FUNC
+    session.configure_standard_waveform(nifgen.Waveform.SINE, 2.0, 0.0, 0.0, 2000000)
+    session.output_enabled = True
+    session.trigger_mode = nifgen.TriggerMode.NIFGEN_VAL_CONTINUOUS
+    session.trigger_source = nifgen.TriggerSource.NIFGEN_VAL_IMMEDIATE
+    with session.initiate():
+        assert session.func_amplitude == 2.0
+        assert session.func_waveform == nifgen.Waveform.SINE
+        assert session.is_done() is False
+
+
+def test_no_waveform_data(session):
```

Comment:
This test seems to be testing the driver, and not the Python bindings. Am I mistaken?
If so, I don't think we need it.